docker/README.md

```diff
@@ -29,7 +29,7 @@ docker run --net=host --name='myriad-resourcemanager' -t \
 
 If you already had a working Vagrant instance, you will need to run `vagrant reload` in order to allow zookeeper and hdfs port-forwarding.
 
-Still having problems connecting to Zookeeper? Be sure that your zkServer and MesosMaster values are correct in the [myriad-config-default.yml](https://github.com/mesos/myriad/blob/phase1/myriad-scheduler/src/main/resources/myriad-config-default.yml) file.
+Still having problems connecting to Zookeeper? Be sure that your zkServer and MesosMaster values are correct in the [myriad-config-default.yml](../myriad-scheduler/src/main/resources/myriad-config-default.yml) file.
 
```

Comment:
Nit, I would change this to:
If you are still having problems connecting to Zookeeper, be sure ...